assets/scss/pages/page.scss

```diff
@@ -13,11 +13,16 @@
 .p-common__title {
```

Comment:
Hm, I'm not sure if I like this approach. What's the reason for using new "pages" level instead of let's say "object"? In `assets/scss/theme.scss` you have 

```
@import 'settings/settings';
@import 'tools/tools';
@import 'generic/generic';
@import 'elements/elements';
@import 'vendor/vendor';
@import 'objects/objects';
@import 'components/components';
@import 'utilities/utilities';
@import 'widgets/widgets';
@import 'pages/pages';
```

Where the idea was to start with most generic and broader style scopes and narrow down the focus/scope to very specific elements/properties (therefore the inverted triangle in ITCSS). But now, after pretty narrow `utilities`, you are importing rather broad `widgets` and even more broad `pages`? 

I know it's not strictly related to this MR, but maybe it's worth to revisit ? :)

Reply:
@aleksandraworhacz let's keep this as is for now and keep it in mind later on. Marcin didn't tell me anything about this usage when I was working on it. Since we are almost at the end of the first demo I don't want to change this and spend more time with it. After the demo we can regroup see what we can improve from this first stage of development and get approval (because i think that if we change this now it will take more time to change)

also, let's say we work on this in the future. how would you do it? remove widgets and pages and then just create component scss files for every component on any page and widget? so basically just focusing on components and not seperating them by pages

Reply:
> so basically just focusing on components and not seperating them by pages

Exactly. Focusing on pages/widgets instead of components will give us a headache when we end up with much of "look alike" code :/